.github/workflows/code-review.yml

```
@@ -17,9 +17,109 @@ jobs:
+        # cat diff.txt
+
+        git_diff=$(cat diff.txt)
+        echo "DIFF<<EOF" >> $GITHUB_ENV
+        echo "$git_diff" >> $GITHUB_ENV
+        echo "EOF" >> $GITHUB_ENV
+        
+        echo "환경변수 확인"
+        echo ${{env.DIFF}}
+
+    - name: review to pull request
+      uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)